ng2-bootstrap.ts

```diff
@@ -3,7 +3,7 @@ export * from  './components/alert';
 export * from  './components/buttons';
 export * from  './components/carousel';
 export * from  './components/collapse';
-// export * from  './components/datepicker/index';
+export * from  './components/datepicker/index';
```

Comment:
@amansharma0091 while merging I have applied some export shift, you will see in my commit
I want to achieve one module per component later